kafka-quickstart/processor/src/test/java/org/acme/kafka/processor/QuoteProcessorTest.java

```diff
@@ -0,0 +1,63 @@
+package org.acme.kafka.processor;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import java.time.Duration;
+import java.util.Collections;
+import java.util.Map;
+import java.util.Properties;
+import java.util.UUID;
+
+import javax.inject.Inject;
+
+import org.acme.kafka.model.Quote;
+import org.apache.kafka.clients.consumer.ConsumerConfig;
+import org.apache.kafka.clients.consumer.ConsumerRecords;
+import org.apache.kafka.clients.consumer.KafkaConsumer;
+import org.apache.kafka.clients.producer.KafkaProducer;
+import org.apache.kafka.clients.producer.ProducerRecord;
+import org.apache.kafka.common.serialization.StringDeserializer;
+import org.apache.kafka.common.serialization.StringSerializer;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import io.quarkus.kafka.client.serialization.ObjectMapperDeserializer;
+import io.quarkus.test.junit.QuarkusTest;
+import io.smallrye.common.annotation.Identifier;
+
+@QuarkusTest
+public class QuoteProcessorTest {
+
+    @Inject
+    @Identifier("default-kafka-broker")
+    Map<String, Object> kafkaConfig;
+
+    KafkaProducer<String, String> quoteRequestProducer;
+    KafkaConsumer<String, Quote> quoteConsumer;
+
+    @BeforeEach
+    void beforeAll() {
+        System.out.println(kafkaConfig);
+        quoteConsumer = new KafkaConsumer<>(consumerConfig(), new StringDeserializer(), new ObjectMapperDeserializer<>(Quote.class));
+        quoteRequestProducer = new KafkaProducer<>(kafkaConfig, new StringSerializer(), new StringSerializer());
+    }
+
+    Properties consumerConfig() {
+        Properties properties = new Properties();
```

Comment:
unrelated to the review, but it's where I believe we can improve our testing support. We could "automagically" compute this configuration from the producer config.

Reply:
Alternatively, we could expose the managed Kafka consumer/producer/admin as bean.